Cargo.lock

```diff
@@ -448,6 +448,18 @@ dependencies = [
  "winapi",
 ]
 
+[[package]]
+name = "filetime"
```

Comment:
A lot of new dependencies :/

Reply:
It comes from new feature flag `gix = { features = ["attributes"], .. }`, which is needed for https://docs.rs/gix/0.58.0/gix/struct.Repository.html#method.filter_pipeline 😢

Reply:
I reviewed how this dependency is used and it seems 'convenience' is the answer. No feature is used that wouldn't be covered by the doing what the crate does.

I added a task to [the `helix` tracking issue](https://github.com/Byron/gitoxide/issues/637) to represent that possibility, and help would definitely be welcome.